keras_hub/src/models/retinanet/retinanet_image_converter.py

```diff
@@ -10,25 +12,36 @@ class RetinaNetImageConverter(ImageConverter):
 
     def __init__(
         self,
-        image_size=None,
```

Comment:
This seems suspicious. Why are we removing these args? Keep these unaltered, add `bounding_box_format` after.

Reply:
Oh, this was by mistake sorry about this.